components/google-cloud/dependencies.py

```diff
@@ -0,0 +1,34 @@
+# Lint as: python3
```

Comment:
IIRC the linter is now default to Python3 so this line is redundant, right?

Reply:
good point, removed across all the files.